package/base-files/files/lib/upgrade/nand.sh

```diff
@@ -229,22 +245,17 @@ nand_upgrade_ubinized() {
 	local ubi_file="$1"
```

Comment:
@dangowrt ok Dan, we got a problem: i *almost* didn't change `nand_upgrade_ubinized` at all! `nand_upgrade_ubinized` doesn't even call `nand_upgrade_prepare_ubi`, so ubiblock devices have nothing to with it.